hs_tools_resource/models.py

```diff
@@ -106,11 +118,7 @@ def update(cls, element_id, **kwargs):
         if 'supported_res_types' in kwargs:
             meta_instance.supported_res_types.clear()
             for res_type_str in kwargs['supported_res_types']:
```

Comment:
I can see here you are looping through res_types list. I suggest moving the for loop to the created function (i.e. add_supported_res_type)